guide/getting-started/13-software.md

```diff
@@ -0,0 +1,166 @@
+
+## Payment processors
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/software/point-of-sale-example.jpg"
+   retina = "/assets/images/guide/getting-started/software/point-of-sale-example@2x.jpg"
+   alt-text = "Illustrative interface for an online shop"
+   caption = "Payment processors help merchants to sell good and manage inventory and accounting."
+   width = 400
+   height = 541
+   layout = "float-right-desktop"
+%}
+
+Making purchases with bitcoin is great, as long as we have stores to buy things from. Payment processing applications offer easy-to-use online stores and point-of-sale experiences, paired with more complex feature sets for merchants to manage their inventory, accounting, shipping, and integrations with other services.
+
+</div>
+
+
+## Exchanges
+
+<div class="center" markdown="1">
+
+Exchanges allow for swapping between currencies and usually fall in three general categories. Consumer-focused exchanges allow for users to make trades directly with the exchange. This is ideal for occasional use.
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/software/exchange-example.jpg"
+   retina = "/assets/images/guide/getting-started/software/exchange-example@2x.jpg"
+   alt-text = "Illustrative interface for exchanging bitcoin"
+   caption = "Exchanges allow for trading between different currencies."
+   width = 400
+   height = 388
+   layout = "float-right-desktop"
+%}
+
+Exchanges focused on investors and traders are marketplaces that typically list many different cryptocurrencies and allow users trade with each other by to submit buy and sell requests on currency pairs. These products are usually for  experienced, regular users and filled with endless ways to analyze price movements and complex trading features.
+
+In between these first two categories fall decentralized exchanges. They may be focused on occasional use or trading, but are always based on an open marketplace without a central authority.
+
+</div>
+
+## Nodes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/software/node-ui-example.jpg"
+   retina = "/assets/images/guide/getting-started/software/node-ui-example@2x.jpg"
+   alt-text = "Illustrative interface for node software"
+   caption = "Node applications connect to and participate in the bitcoin network."
+   width = 400
+   height = 426
+   layout = "float-right-desktop"
+%}
+
+A node connects to and participates in the bitcoin network to download and exchange transaction data. Some wallets have built-in nodes and some node software also supports wallet functionality. The minimum implementation of a node however is that of syncing data and allowing other software a user has to access it.
```

Comment:
> The minimum implementation of a node however is that of syncing data and allowing other software a user has to access it.
	
Could be me, but I don't quite understand this sentence.  _is that of syncing data_ is confusing to me.